src/ctree-app/ctree-landing.html

```diff
@@ -533,33 +594,59 @@
           <p>W</p>
         </div>
         <div id="features" class="section-styling">
-          <div class= "center-div">
-            <h1 align="center">Features</h1>
-            <hr id="features-horizontal-line"/>
-            <!-- grid layout to be added here with icons and descriptions. -->
-          </div>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-        </div>
+          <h1 align="center">Features</h1>         
+          <!-- grid layout to be added here with icons and descriptions. -->
+          <div class="features-content">
+            <div class="collaboration-search-col">
+              <div class="collaboration-section">
+                <div class="image">
+                  <iron-icon icon="collaboration"></iron-icon>
+                </div>
+                <div class="collaboration-text">
+                  <h3>Collaboration</h3>
+                  <p>
+                    Connect with like-minded passionate individuals to solve problems for a cause.
+                  </p>
+                </div>
+              </div> <!-- end of collaboration section -->
+              <div class="search-section">
+                <div class="image">
+                  <iron-icon icon="search"></iron-icon>
+                </div>
+                <div class="search-text">
+                  <h3>Search</h3>
+                  <p>
+                    Discover non-profits and other causes doing good for the world!
+                  </p>
+                </div>
+              </div> <!-- end of search section -->
+            </div> <!-- end of collaboration-search column -->
+            <div class="share-review-col">
+              <div class="share-section">
+                <div class="image">
+                  <iron-icon icon="share"></iron-icon>
+                </div>
+                <div class="share-text">
+                  <h3>Share</h3>
+                  <p>
+                    Promote other helpful thoughts and discussions to avoid seeing the same answers over and over again
+                  </p>
+                </div>
+              </div> <!-- end of share section -->
+              <div class="review-section">
+                <div class="image">
+                  <iron-icon class="review" icon="review"></iron-icon>
+                </div>
+                <div class="review-text">
```

Comment:
I'm not sure about separate class names for each item like this.  It's probably good web dev practice, but I don't see why each would need to have its own styling, so it seems unnecessary to do this.  It'll be a moot point if you convert these to Polymer components.

Reply:
You are right. I deleted the class names that were not needed in my css. Yet, the divs and class names that were needed for the flex layout, i had to keep in order to distinguish from the parents/child containers.